go/vt/vttablet/tabletmanager/vreplication/engine.go

```diff
@@ -68,6 +68,7 @@ func init() {
 	allddls := append([]string{}, binlogplayer.CreateVReplicationTable()...)
 	allddls = append(allddls, binlogplayer.AlterVReplicationTable...)
 	allddls = append(allddls, createReshardingJournalTable, createCopyState)
+	allddls = append(allddls, binlogplayer.AlterVReplicationTableSourceToBlob...)
```

Comment:
This can be removed once query is moved into `AlterVReplicationTable`

Reply:
done